setup/date_range_account/odoo/addons/date_range_account

```diff
@@ -0,0 +1 @@
+../../../../date_range_account
```

Comment:
Not present in 14.0 nor 16.0, it may not be strictly needed. It also looks like some leftover file.